adapters/concert/concert.go

```diff
@@ -0,0 +1,165 @@
+package concert
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"net/http"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+const adapterVersion = "1.0.0"
+
+// Builder builds a new instance of the Concert adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	bidderImpExt, err := getBidderExt(request.Imp[0])
```

Comment:
Just re-checking - Are you sure to use just first impression and ignore others. If you wanna consider sending separate http request to your bidder server, you may choose to do so.

Reply:
Yes. As of now, this is only used for extracting the `partnerId` which should be the same across all Imp's.